src/Symfony/Component/Form/Tests/AbstractBootstrap3LayoutTest.php

```diff
@@ -214,7 +214,7 @@ public function testSingleChoice()
         $form = $this->factory->createNamed('name', 'Symfony\Component\Form\Extension\Core\Type\ChoiceType', '&a', array(
             'choices' => array('Choice&A' => '&a', 'Choice&B' => '&b'),
             'multiple' => false,
```

Comment:
if `widget` is set as `select`, `multiple can be omitted too IMO.
You should maybe add a test without `widget` to see it falls back on `select`.